### ISSUE TITLE
Suggestion: Show MP amount by default

### ISSUE THREAD

**Issue by pyroll** [gameboy9/FF6KefkaRush#2]
You can have seeds where you start with someone like Strago who is able to cast Lore magic, but you can't see how much MP he has. I assume it's tied to when you first acquire Espers in vanilla.

**Comment by gameboy9** [comment]
You are correct; it is tied to when you first acquire a Magicite.  Therefore, to accommodate your request, we would have to award a Magicite at the beginning of the game.

That would be a nice option, but not until I hammer out all of the issues.

**Comment by gameboy9** [comment]
The first part of the request is complete due to Strago's Lores.(we didn't know how much MP Strago had when casting a Lore)  The second part, an option of how many magicite to start with at the beginning of the game, I plan to complete by 1.0.
